splink/comparison_library_utils.py

```diff
@@ -60,3 +60,64 @@ def datediff_error_logger(thresholds, metrics):
         raise ValueError("\n\n".join(error_logger))
 
     return
+
+
+def distance_threshold_comparison_levels(
+    self,
+    col_name: str,
+    distance_function_name: str,
+    distance_threshold_or_thresholds,
+    higher_is_more_similar: bool = True,
+    m_probability_or_probabilities_thres: list = None,
+):
+    thresholds = ensure_is_iterable(distance_threshold_or_thresholds)
+    threshold_comparison_levels = []
+
+    if m_probability_or_probabilities_thres is None:
+        m_probability_or_probabilities_thres = [None] * len(thresholds)
+    m_probability_or_probabilities_thres = ensure_is_iterable(
+        m_probability_or_probabilities_thres
+    )
+
+    for thres, m_prob in zip(thresholds, m_probability_or_probabilities_thres):
+        if distance_function_name == "levenshtein":
+            distance_function_name = self._levenshtein_name
+            higher_is_more_similar = False
```

Comment:
Can we bake `higher_is_more_similar` into the base class? 

I'll check this code in a little bit more detail and work out if I'm being obtuse.

Reply:
Looking at it, we can't because of where this is being used. I think it's fine.